content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,416 @@
+---
+weight: 50
+title: Command Reference
+description: "Command reference for the Crossplane CLI"
+---
+
+<!-- vale Google.Headings = NO -->
+The `crossplane` CLI provides utilities to make using Crossplane easier. 
```

Comment:
Does "easier" go against the style/contributing guide here?

If so, perhaps "more convenient"?

Reply:
No violation. The rule is about saying something is easy (i.e., if you don't understand you're dumb). In this case we're talking about a relative change.